numcosmo/math/ncm_stats_dist_nd_kde_studentt.c

```diff
@@ -294,11 +323,11 @@ ncm_stats_dist_nd_kde_studentt_class_init (NcmStatsDistNdKDEStudenttClass *klass
                                                       1, G_MAXUINT, 200,
                                                       G_PARAM_READWRITE | G_PARAM_CONSTRUCT | G_PARAM_STATIC_NAME | G_PARAM_STATIC_BLURB));
   g_object_class_install_property (object_class,
-                                   PROP_FREEDOM,
-                                   g_param_spec_uint("FREEDOM",
+                                   PROP_NU,
+                                   g_param_spec_double ("nu",
                                                      NULL,
-                                                     "Freedom value of the function",
-                                                      1, G_MAXUINT, 200,
+                                                     "nu value of the function",
+                                                      1.0, G_MAXDOUBLE, 400.0,
```

Comment:
I don't know which should be the maximum value of this property. 400.0 is a random number.